bgpd/bgp_routemap.c

```diff
@@ -6305,6 +6304,15 @@ DEFUN_YANG (no_set_aspath_prepend,
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+ALIAS_YANG (no_set_aspath_prepend,
+            no_set_aspath_prepend_as_cmd,
+            "no set as-path prepend ASNUM...",
```

Comment:
Could you please provide some UT for these options.
I am unclear about the no set as-path prepend <id> as your line 6291 syntax removed it.